gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -155,6 +155,12 @@ def add_arguments(self, parser):
               '[EXPERIMENTAL]'
              ).format(sample_info_table_schema_generator.TABLE_SUFFIX))
 
+    parser.add_argument(
+        '--include_file_in_sample_id',
```

Comment:
In general I'd rather we use flags that convey a meaning rather than a specific implementation detail, for two specific reasons:
1. It might be more meaningful to the user
2. We can reuse this information in other contexts

In this case, the real question is whether a single sample spans multiple files.  So something like --samples_span_multiple_files might be more appropriate.

What do you think?

Reply:
Thanks Aaron, I modified the flag name.